File: scanner/corpora/javascript/negative/no_crypto.js
/** Utility module with no cryptographic operations. */

function formatData(items) {
    return items.map(i => i.toUpperCase());
}

class DataStore {
    constructor() {
        this.data = [];
    }
    add(item) {
        this.data.push(item);
    }
    getAll() {
        return [...this.data];
    }
}

function calculateSum(a, b) {
    return a + b;
}

module.exports = { formatData, DataStore, calculateSum };

File: scanner/corpora/python/negative/no_crypto.py
"""Utility module with no cryptographic operations."""

def process_data(items):
    results = []
    for item in items:
        results.append(item * 2)
    return results

class DataProcessor:
    def __init__(self, name):
        self.name = name

    def transform(self, data):
        return data.upper()

def calculate_sum(a, b):
    return a + b
